[@types/semantic-ui] error when try to add param `type: "category"` on search param

Hello, i got error when try to add param `type: "category"` on search param

- [x] I tried using the `@types/semantic-ui` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] Mention authors: [Leonard Thieu](https://github.com/leonard-thieu) @leonard-thieu 

```
function setupSearchInput(): void {
    let url: string = API_URL + '/read-all-search.php?q={query}';
    let apiSettings: SemanticUI.ApiSettings = {
        url: url
    };
    let fieldsSettings: SemanticUI.Search.FieldsSettings = {
        results: 'results',
        title: 'nama'
    }
    let searchSettings: SemanticUI.SearchSettings = {
        minCharacters: 1,
        fields: fieldsSettings,
        apiSettings: apiSettings,
        onSelect: function (this: JQuery, result: any, response: any) {
            onSelectSearchResult(this, result, response);
        },
        type: "category"
    };
    $('.ui.search').search(searchSettings);
}
```

Error message - Visual Studio Code:
```
[ts]
Type '{ minCharacters: number; fields: (Pick<_Impl, "results"> & Partial<Pick<_Impl, "action" | "catego...' is not assignable to type 'Param'.
Object literal may only specify known properties, and 'type' does not exist in type 'Param'.
```

Error message - `webpack`:
```
ERROR in ./src/app_map.ts
(1289,9): error TS2322: Type '{ minCharacters: number; fields: (Pick<_Impl, "results"> & Partial<Pick<_Impl, "action" | "catego...' is not assignable to type 'Param'.
  Object literal may only specify known properties, and 'type' does not exist in type 'Param'.
```

Screenshot - file `app_map.ts` (my project file):
- [https://imgur.com/BE63rkJ](https://imgur.com/BE63rkJ)
- [https://imgur.com/nF5qYrK](https://imgur.com/nF5qYrK)

Screenshot - Semantic UI Search docs:
- [https://imgur.com/igthuhu](https://imgur.com/igthuhu)
- [https://imgur.com/V4Z9EUO](https://imgur.com/V4Z9EUO)

Thanks

--- a/types/semantic-ui-search/global.d.ts
+++ b/types/semantic-ui-search/global.d.ts
@@ -107,6 +107,7 @@ declare namespace SemanticUI {
 
     namespace SearchSettings {
         type Param = (Pick<_Impl, 'apiSettings'> |
+            Pick<_Impl, 'type'> |
             Pick<_Impl, 'minCharacters'> |
             Pick<_Impl, 'transition'> |
             Pick<_Impl, 'duration'> |
@@ -142,6 +143,8 @@ declare namespace SemanticUI {
             Partial<Pick<_Impl, keyof _Impl>>;
 
         interface _Impl {
+            type: string;
+
             // region Behavior
 
             /**

--- a/types/semantic-ui-search/semantic-ui-search-tests.ts
+++ b/types/semantic-ui-search/semantic-ui-search-tests.ts
@@ -46,6 +46,7 @@ function test_search() {
     });
     // $ExpectType JQuery<HTMLElement>
     $(selector).search({
+        type: 'type',
         apiSettings: {
             on: 'on',
             cache: true,

